### PR TITLE
Update organizations documentation in the server admin guide

### LINKED ISSUE #33199: [Organizations] Review documentation for Keycloak 26
### Description

Organizations will be a supported feature in Keycloak 26. As such, we need to ensure the documentation is up-to-date with all the changes made since its introduction in Keycloak 25.

### PR DESCRIPTION
**Pull request by sguilhen**
Closes #33199

### CODE REVIEW

File: docs/documentation/server_admin/topics/organizations/authenticating-members.adoc [hunk 1]
```diff
@@ -30,29 +30,35 @@ boundaries through an identity provider.
...
+In addition to identifying the user once the username is provided, the identity-first login is also responsible for:
 
-* Match an email domain to an organization
-* Decide if the authentication flow should continue or not if an account already exists for the username provided
-* Decide how the user should be authenticated depending on how the domains and the identity providers are configured to an organization
-* Seamlessly authenticate users through an identity provider associated with an organization if the email domain matches the domain set to the identity provider
+* Matching an email domain to an organization;
+* Deciding if the authentication flow should continue or not if an account already exists for the username provided;
+* Deciding how the user should be authenticated depending on how the domains and the identity providers are configured to an organization;
+* Seamlessly authenticate users through an identity provider associated with an organization if the email domain matches the domain set to the identity provider.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/authenticating-members.adoc:38]:
```suggestion
* Matching an email domain to an organization.
* Deciding if the authentication flow should continue or not if an account already exists for the username provided
* Deciding how the user should be authenticated depending on how the domains and the identity providers are configured to an organization
* Seamlessly authenticating users through an identity provider associated with an organization if the email domain matches the domain set to the identity provider
```

Comment (andymunro) [reply docs/documentation/server_admin/topics/organizations/authenticating-members.adoc:38]: No need for semi-colons or periods if a bullet is a partilal sentence

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 2]
```diff
@@ -32,11 +39,41 @@ The identity provider you want to link to the organization. An identity provider
...
 
-Once linked to an organization, the identity provider can be managed just like any other in a realm by accessing the *Identity Providers* section in the menu.
-However, the options herein described are only available when managing the identity provider in the scope of an organization.
+Once linked to an organization, the identity provider can be managed just like any other in a realm by accessing the *Identity Providers* section in the menu. However, the options herein described are only available when managing the identity provider in the scope of an organization. The only exception is the
+ *Hide on login page* option that is present here for convenience.
+
+== Editing a linked identity provider
+
+You can edit any of the organization-related settings of a linked identity provider at any time. go to the *Identity providers* tab in the organization, locate the provider in the list, then click the action button (with three dots) at the end of the line and select *Edit*.
+
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:54]:
```suggestion
You can edit any of the organization-related settings of a linked identity provider at any time. 
```

Comment (andymunro) [reply docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:54]: The procedure already has the steps.

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 3]
```diff
@@ -32,11 +39,41 @@ The identity provider you want to link to the organization. An identity provider
...
+.Procedure
+
+. Go to the *Identity providers* tab in the organization settings
+. Locate the *identity provider* in the list (you can use the search capabilities for that)
+. Click on the action button (three dots) at the end of the line
+. Click *Edit*
+. Make the necessary changes
+. Click *Save*
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:62]:
```suggestion
. In the organization settings, go to the *Identity providers* tab.
. Locate the *identity provider* in the list.
+
You can use the search option for this step.
. Click the action button (three dots) at the end of the line.
. Click *Edit*.
. Make the necessary changes.
. Click *Save*.
```

Comment (andymunro) [reply docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:62]: We include a period for a complete sentence and use "click" not "click on"

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 4]
```diff
@@ -32,11 +39,41 @@ The identity provider you want to link to the organization. An identity provider
...
+
+.Editing linked identity provider
+image:images/organizations-edit-identity-provider.png[alt="Editing linked identity provider"]
+
+== Unlinking an identity provider from an organization
+
+When an identity provider is unlinked from an organization, it continues to be available as a realm-level provider that is not associated with an organization anymore. If you want to delete the unlinked provider, it has to be done through the *Identity Providers* section in the menu.
+
+.Procedure
+
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:72]:
```suggestion
When an identity provider is unlinked from an organization, it remains available as a realm-level provider that is no longer ssociated with an organization. If you want to delete the unlinked provider, use the *Identity Providers* section in the menu.

.Procedure

```

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 5]
```diff
@@ -32,11 +39,41 @@ The identity provider you want to link to the organization. An identity provider
...
+. Go to the *Identity providers* tab in the organization settings
+. Locate the *identity provider* in the list (you can use the search capabilities for that)
+. Click on the action button (three dots) at the end of the line
+. Click *Unlink provider*
+
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:77]:
```suggestion
.  In the organization settings, go to the *Identity providers* tab.
. Locate the *identity provider* in the list.
.+
You can use the search capabilities for this step.
. Click the action button (three dots) at the end of the line.
. Click *Unlink provider*.

```

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 6]
```diff
@@ -32,11 +39,44 @@ The identity provider you want to link to the organization. An identity provider
...
-However, the options herein described are only available when managing the identity provider in the scope of an organization.
+Once linked to an organization, the identity provider can be managed just like any other in a realm by accessing the *Identity Providers* section in the menu. However, the options herein described are only available when managing the identity provider in the scope of an organization. The only exception is the
+ *Hide on login page* option that is present here for convenience.
+
+== Editing a linked identity provider
+
+You can edit any of the organization-related settings of a linked identity provider at any time. 
+.Procedure
+
+. In the organization settings, go to the *Identity providers* tab.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:56]:
```suggestion
. In the menu, click *Organizations* and go to the *Identity providers* tab.
```

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 7]
```diff
@@ -32,11 +39,44 @@ The identity provider you want to link to the organization. An identity provider
...
+.Editing linked identity provider
+image:images/organizations-edit-identity-provider.png[alt="Editing linked identity provider"]
+
+== Unlinking an identity provider from an organization
+
+When an identity provider is unlinked from an organization, it remains available as a realm-level provider that is no longer ssociated with an organization. To delete the unlinked provider, use the *Identity Providers* section in the menu.
+
+.Procedure
+
+.  In the organization settings, go to the *Identity providers* tab.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:74]:
```suggestion
. In the menu, click *Organizations* and go to the *Identity providers* tab.
```

File: docs/documentation/server_admin/topics/organizations/managing-organization.adoc [hunk 8]
```diff
@@ -1,42 +1,49 @@
...
 An organization has the following settings:
 
 Name::
 A user-friendly name for the organization. The name is unique within a realm.
 
 Alias::
-An alias for this organization, used to reference the organization internally. The alias is unique within a realm.
-If not set, the value is the same as the organization name. The alias cannot change afterwards.
+
+An alias for this organization, used to reference the organization internally. The alias is unique within a realm and must be URL-friendly, so characters not usually allowed in URLs won't be allowed in the alias. If not set, {project_name} will attempt to use the name as the alias. If the name is not URL-friendly, you will get an error and will be asked to specify an alias. Once defined, the alias cannot be changed afterwards.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-organization.adoc:43]:
```suggestion
An alias for this organization, used to reference the organization internally. The alias is unique within a realm and must be URL-friendly, so characters not usually allowed in URLs will not be allowed in the alias. If not set, {project_name} will attempt to use the name as the alias. If the name is not URL-friendly, you will get an error and will be asked to specify an alias. Once defined, the alias cannot be changed afterwards.
```

File: docs/documentation/server_admin/topics/organizations/managing-organization.adoc [hunk 9]
```diff
@@ -52,35 +59,33 @@ Once you create an organization, you can manage the additional settings that are
...
-However, the unmanaged members of an organization are still able to authenticate to the realm as they are also realm users, but
-tokens will not hold metadata about their relationship with an organization that is disabled.
+However, the unmanaged members of an organization are still able to authenticate to the realm as they are also realm users, but tokens will not hold metadata about their relationship with an organization that is disabled.
 
 For more details about managed and unmanaged users, see <<_managed_unmanaged_members_,Managed and unmanaged members>> section.
 
-== Remove an organization
+== Deleting an organization
+
+To delete an organization, you can click the *Delete* action for the corresponding organization in the listing page or when editing an organization.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-organization.adoc:82]:
```suggestion
To delete an organization, click the *Delete* action for the corresponding organization in the listing page or when editing an organization.
```

Comment (andymunro) [reply docs/documentation/server_admin/topics/organizations/managing-organization.adoc:82]: Generally, a "you can do x" can be replace with "do x" which is more concise.

File: docs/documentation/server_admin/topics/organizations/mapping-organization-claims.adoc [hunk 10]
```diff
@@ -3,22 +3,30 @@
...
 ```
 
 The organization claim can be used by clients (for example, from ID Tokens) and resource servers (for example, from access tokens)
 to authorize access to protected resources based on the organization where the user is a member.
 
-The `organization` scope is a built-in optional client scope at the realm.  Therefore, this scope is added to any client created
-in the realm, by default.
+The `organization` scope is a built-in optional client scope at the realm.  Therefore, this scope is added to any client created in the realm by default. It also defines the `Organization Membership` mapper that controls how the organization membership information is mapped to the tokens.
+
+By default the organization attributes are not included in the organization claim. To include the attributes in the claim, edit the mapper and switch the *Add organization attributes* setting to on.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/mapping-organization-claims.adoc:26]:
```suggestion
By default, the organization attributes are not included in the organization claim. To include the attributes in the claim, edit the mapper and enable *Add organization attributes*.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good. A few more changes.

File: docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc [hunk 11]
```diff
@@ -32,11 +39,44 @@ The identity provider you want to link to the organization. An identity provider
...
+
+== Unlinking an identity provider from an organization
+
+When an identity provider is unlinked from an organization, it remains available as a realm-level provider that is no longer ssociated with an organization. To delete the unlinked provider, use the *Identity Providers* section in the menu.
+
+.Procedure
+
+. In the menu, click *Organizations* and go to the *Identity providers* tab.
+. Locate the *identity provider* in the list.
+.+
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:76]:
```suggestion
+
```

Comment (andymunro) [reply docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc:76]: I added the period by mistake.

File: docs/documentation/server_admin/topics/organizations/mapping-organization-claims.adoc [hunk 12]
```diff
@@ -3,22 +3,30 @@
 =  Mapping organization claims
 [role="_abstract"]
 To map organization-specific claims into tokens, a client needs to request the *organization* scope when sending
-authorization requests to the server. When authenticating in the context of an organization, clients can request the `organization` scope to map to tokens information
-about the organizations the user is a member.
+authorization requests to the server. When authenticating in the context of an organization, clients can request the `organization` scope to map information
+about the organizations the user is a member of to tokens.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/organizations/mapping-organization-claims.adoc:7]:
```suggestion
authorization requests to the server. When authenticating in the context of an organization, clients can request the `organization` scope to map information
about the organizations where the user is a member.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
A couple of final changes, @sguilhen

**Review by andymunro — APPROVED** [review]
APPROVED, @sguilhen!

**Comment by sguilhen** [comment]
Please squash the commits when merging.

**Comment by ahus1** [comment]
@martin-kanis / @vramik - can someone from the team review the contents of the docs that they match the implementation? It is great to have Andrew reviewing it from a technical writing perspective, but for these changes another review from the team would be great. Thanks!

cc: @sguilhen

**Review by martin-kanis — APPROVED** [review]
Thanks @sguilhen, LGTM.

**Comment by sguilhen** [comment]
@ahus1 When you find some time to merge, the reviews are in place!

**Review by ahus1 — APPROVED** [review]
Approving based on previous reviews

**Pull request merged**